### DISCUSSION TITLE
[Question]: should discussion mode trigger listening automatically?

### DISCUSSION THREAD

**Discussion by lio1204** [danny-avila/LibreChat#3921] [Q&A]
### What is your question?

Hi there,
First of all, congrats on the amazing work on LibreChat. Definitely one of the best projects around on this matter, and I do intend to participate in its development!
Had a first question related to my use case (which is very much audio based): I did activate external STT & TTS and its working fine. I can read last reply aloud automatically. However after the last reply is read through audio, shouldn't it trigger automatically a new audio capture to listen to what user has to say? That would be a "true discussion" according to me. 

### More Details

Just wondering if it's not the case on purpose for now, or if I missed something.
Thanks in advance for your reply!

### What is the main subject of your question?

Documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by geodanchev** [comment]
I also would like to have this feature. I will be great. Chat GPT is working this way and it is awesome.
